draft-ietf-masque-h3-datagram.md

```diff
@@ -103,6 +103,26 @@ hop, but the context ID will remain the same. Context IDs are opaque to
 intermediaries.
 
 
+## Datagram Formats {#datagram-formats}
+
+When an endpoint registers a datagram context (or the lack of contexts), it
+communicates the format (i.e., the semantics and encoding) of datagrams sent
+using this context. This is acccomplished by sending a Datagram Format Type as
+part of the registration capsule, see {{register-capsule}} and
+{{register-no-context-capsule}}. This type identifier is registered with IANA
+(see {{iana-format-types}}) and allows applications that use HTTP Datagrams to
+indicate what the content of datagrams are. Registration capsules carry a
+Datagram Format Data field which allows sending some additional information
```

Comment:
Following from the above comment, since it sounds like datagrams always have a format, maybe it would be better to call this "Datagram Format Additional Data" to clearly mark it as separate to actual datagram data?

Reply:
Good idea, done.